contracts/test/Misbehaviour.t.sol

```diff
@@ -42,14 +39,13 @@ contract SP1ICS07MisbehaviourTest is SP1ICS07TendermintTest {
 
         submitMsg = abi.decode(fixture.submitMsg, (IMisbehaviourMsgs.MsgSubmitMisbehaviour));
         output = abi.decode(submitMsg.sp1Proof.publicValues, (IMisbehaviourMsgs.MisbehaviourOutput));
-        env = output.env;
     }
 
     function test_ValidDoubleSignMisbehaviour() public {
-        setUpMisbehaviour("misbehaviour_double_sign_fixture.json");
+        setUpMisbehaviour("misbehaviour_double_sign-plonk_fixture.json");
```

Comment:
super duper nit, but why is this file not named "misbehaviour_double_sign_fixture-plonk.json" to match the other files? :D

Reply:
Because the way e2e's generate fixtures by appending `_fixture` to the end of the fixture name. I think this is fine because misbehavior fixtures are generated by a different process. So it reflects some implementation detail

Reply:
Fair enough :)